ui/src/App.tsx

```diff
@@ -49,6 +48,16 @@ export default function App() {
     };
   }, [token, channelSocket, userSocket]);
 
+  React.useEffect(() => {
```

Comment:
@hskowron is that correct to set it here ?

Reply:
Actually, it has no effect there. This useEffect will only be entered before we have logged in so it will never have a token to initialize the socket with. Unless we have a way of detecting that the token is now in localStorage we just have to have a useEffect like that on every view.